### PR TITLE
Reading assets path from merge-assets task

### PR DESCRIPTION
**Pull request by menny**

### CODE REVIEW

File: paparazzi-gradle-plugin/src/main/java/app/cash/paparazzi/gradle/PaparazziPlugin.kt [hunk 1]
```diff
@@ -46,6 +46,7 @@ class PaparazziPlugin : Plugin<Project> {
           "preparePaparazzi${variantSlug}Resources", PrepareResourcesTask::class.java
       ) { task ->
         task.mergeResourcesOutput.set(variant.mergeResourcesProvider.flatMap { it.outputDir })
+        task.assetsDirectory.set(variant.mergeAssetsProvider.flatMap { it.outputDir })
```
Comment (jrodbx) [inline paparazzi-gradle-plugin/src/main/java/app/cash/paparazzi/gradle/PaparazziPlugin.kt:49]: `task.mergeAssetsOutput` for consistency

File: paparazzi/src/main/java/app/cash/paparazzi/Environment.kt [hunk 2]
```diff
@@ -21,11 +21,9 @@ data class Environment(
   val platformDir: String,
   val appTestDir: String,
   val resDir: String,
+  val assetsDir: String,
   val packageName: String,
-  val compileSdkVersion: Int
-) {
-  val assetsDir = "$appTestDir/src/main/assets/"
-}
+  val compileSdkVersion: Int)
```
Comment (jrodbx) [inline paparazzi/src/main/java/app/cash/paparazzi/Environment.kt:27]: could the assets ever be in a location other than `$appTestDir/src/main/assets`?

**Review by jrodbx — CHANGES_REQUESTED** [review]

File: paparazzi/src/main/java/app/cash/paparazzi/Environment.kt (hunk 2 above)
Comment (menny) [reply paparazzi/src/main/java/app/cash/paparazzi/Environment.kt:27]: yes: either you are using another variant, or somehow decided to change the source-set folder in your gradle setup.
Also, `/src/main/assets` is not a _merged_ assets folder (as-in, inherites assets from dependencies).

**Review by jrodbx — APPROVED** [review]

**Comment by jrodbx** [comment]
Resolved merge conflict and merged!

**Comment by jrodbx** [comment]
Resolved merge conflict and merged!

**Pull request closed without merging**